services/core/shared/transactionStatistics.js

```diff
@@ -13,79 +13,122 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { Logger } = require('lisk-service-framework');
-
-const logger = Logger();
-
 const moment = require('moment');
+
+const config = require('../config');
 const transactionStatisticsQueue = require('./transactionStatisticsQueue');
-const { getDatabase, dbQueries } = require('./postgres');
+const getDbInstance = require('./pouchdb');
 
-const transformParamsForDb = ({ dateFrom, dateTo, ...rest }) => ({
-	...rest,
-	dateFrom: moment(dateFrom).format('YYYY-MM-DD'),
-	dateTo: moment(dateTo).format('YYYY-MM-DD'),
-});
+const getStatsTimeline = async params => {
+	const db = await getDbInstance(config.db.collections.transaction_statistics.name);
 
-const database = getDatabase();
+	const result = await db.find({
+		selector: {
+			date: {
+				$gte: params.dateFrom.toISOString(),
+				$lte: params.dateTo.toISOString(),
+			},
+			// 	WHERE $<dateFrom> <= timestamp AND timestamp <= $<dateTo>
+		},
+	});
```

Comment:
Isn't it better if we write a common function for this and reuse that function wherever needed in the file?

Reply:
Done!